tests/test_browser_model.py

```diff
@@ -36,7 +36,15 @@ def test_close_window(foxpuppet):
 def test_switch_to(foxpuppet, selenium):
     """Test Switch to function"""
     foxpuppet.browser.open_window()
-
     # Switch to originally window opened by pytest
     foxpuppet.browser.switch_to()
     assert foxpuppet.browser.handle == selenium.current_window_handle
+
+
+def test_tracking_protection_shield(foxpuppet, selenium):
+    """Tests if the tracking protection icon displays"""
+    browser = foxpuppet.browser.open_window(private=True)
+    browser.switch_to()
+    assert not browser.navbar.is_tracking_shield_displayed
+    selenium.get('http://cnn.com')
```

Comment:
I'd like to avoid using an external website. We already have a local http server for another test, could you see if we can create some test content that triggers the tracking protection?